src/mobile/yarn.lock

```diff
@@ -6997,9 +6997,16 @@ react-native-markdown-renderer@^3.2.1:
     prop-types "^15.5.10"
     react-native-fit-image "^1.5.2"
 
+<<<<<<< HEAD
+react-native-modal-translucent@^1.1.3:
+  version "1.1.7"
+  resolved "https://registry.yarnpkg.com/react-native-modal-translucent/-/react-native-modal-translucent-1.1.7.tgz#ca2599c0090a7b764c1044a03a13d44e83c95edd"
+  integrity sha512-HMH76pMO+yg55xubjboqlr6TFWo5fHSZtyBBllDPiKcaECrd9Q/gpD/flaFRxgx3Cd8QPLiMh270SBpEDBs1fQ==
+=======
 "react-native-modal-translucent@https://github.com/listenzz/react-native-modal-translucent":
   version "1.1.7"
   resolved "https://github.com/listenzz/react-native-modal-translucent#87942974c2d4b1b405bd08069bcb35d8fdb19959"
+>>>>>>> develop
```

Comment:
I think this is an unresolved merge conflict, if you run `yarn` in `mobile` it should automatically fix it